.github/workflows/ci.yml

```diff
@@ -91,7 +91,7 @@ jobs:
           extra_args: --source ${{ github.event.pull_request.base.sha || 'HEAD~1' }} --origin ${{ github.event.pull_request.head.sha || 'HEAD' }}
 
   build-to-npm:
-    if: github.ref == 'refs/heads/main'
+    # if: github.ref == 'refs/heads/main'
```

Comment:
Should these comments be removed? (Also L135)

Reply:
@MadDataScience ya i just wanted to test the jobs on the PR rather than on merge, will remove!